views/developer.erb

```diff
@@ -88,7 +88,7 @@
 			<li>The maximum size of Docker image that may be staged is 4GB</li>
 			</ul>
 			<h2 class="heading-medium">Backing services</h2>
-			<p>GOV.UK PaaS provides a number of highly available backing services.</p>
+			<p>GOV.UK PaaS provides a number of highly available backing services. Please visit our <a href="https://admin.london.cloud.service.gov.uk/marketplace">marketplace</a> to view more information about these services.</p>
```

Comment:
Suggest extending the link text to "our marketplace" to make the link make more sense in isolation